galaxy_importer/utils/markup.py

```diff
@@ -0,0 +1,123 @@
+# (c) 2012-2019, Ansible by Red Hat
+#
+# This file is part of Ansible Galaxy
+#
+# Ansible Galaxy is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License as published by
+# the Apache Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Ansible Galaxy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License
+# along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
+
+import collections
+import hashlib
+import mimetypes
+import os
+
+import markdown
+import bleach
+from bleach_whitelist import markdown_tags, markdown_attrs
+
+README_NAME = 'README'
+DOCFILE_EXTENSIONS = [
+    '.md',
+]
+DOCFILE_MIMETYPES = {
+    '.md': 'text/markdown',
+}
+DOCFILE_MAX_SIZE = 512 ** 2  # 512 KiB
+
+DocFile = collections.namedtuple(
+    'DocFile', ['name', 'text', 'mimetype', 'hash']
+)
+
+for _ext, _type in DOCFILE_MIMETYPES.items():
+    mimetypes.add_type(_type, _ext)
+
+
+class FileSizeError(Exception):
+    pass
+
+
+def get_readme_doc_file(directory):
```

Comment:
Can you please add docstrings describing what these functions do and return as result.